pkg/hive/manager.go

```diff
@@ -107,11 +106,11 @@ func NewFromConfig(log *logrus.Entry, _env env.Core, restConfig *rest.Config) (C
 	}, nil
 }
 
-func (hr *clusterManager) CreateNamespace(ctx context.Context) (*corev1.Namespace, error) {
+func (hr *clusterManager) CreateNamespace(ctx context.Context, docID string) (*corev1.Namespace, error) {
```

Comment:
one question here, do we need to modify the place where this function is called by adding an extra param? i.e.,

https://github.com/Azure/ARO-RP/blob/f063e9baaf9286781da38276859b81b08a702562/pkg/cluster/hive.go#L20